### PR TITLE
Changed path of sql_workshop files to content > english > sql_workshop

### PR DESCRIPTION
**Pull request by kimtienvu**

### CODE REVIEW

File: content/english/sql_workshop/_index.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+title: "SQL Workshop"
+description: "SQL Workshop Intro"
+date: 2019-07-25T13:24:17-07:00
+prereq: "None"
+difficulty: "Beginner"
+image: ""
+draft: false
+# icon: "fab fa-python"
```
Comment (ozhang22) [inline content/english/sql_workshop/_index.md:9]: Please pick a new icon for this workshop based on: https://forkaweso.me/Fork-Awesome/icons/

File: content/english/sql_workshop/select_command/_activity1.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
+---
+title: "Learn SELECT Command"
+description: "Learn SELECT command"
```
Comment (ozhang22) [inline content/english/sql_workshop/select_command/_activity1.md:3]:
```suggestion
title: "Learn the SELECT Command"
description: "Learn the SELECT command"
```

File: content/english/sql_workshop/_index.md (hunk 1 above)
Comment (ozhang22) [inline content/english/sql_workshop/_index.md:9]: Pick a new icon from: https://forkaweso.me/Fork-Awesome/icons/

Comment (Anvesha4) [reply content/english/sql_workshop/_index.md]: This one looks about right: https://forkaweso.me/Fork-Awesome/icon/database/

File: content/english/sql_workshop/select_command/_activity1.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
```
Comment (Anvesha4) [inline content/english/sql_workshop/select_command/_activity1.md:1]: Looks like it got merged correctly this time! 👍

**Pull request merged**